core/consensus/common/src/import_queue.rs

```diff
@@ -389,28 +333,11 @@ impl<B: BlockT> BlockImporter<B> {
 			};
 		}
 		if let Some(link) = self.link.as_ref() {
-			link.maintain_sync();
+			link.blocks_processed(hashes, has_error);
```

Comment:
`fn blocks_processed` doesn't maintain sync anymore. Where is that call done?

Reply:
The call is made at https://github.com/paritytech/substrate/pull/1818/files#diff-9c1b34b73c6680d46fc07cae4591547fR347 as part of handling `blocks_processed`. I removed the `maintain_sync` method on `Link` and replaced it with `blocks_processed`, which still does the "mainting sync" as part of it...